services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/api/_resource_usage_tracker.py

```diff
@@ -0,0 +1,36 @@
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Depends
+from fastapi_pagination.api import create_page
+from fastapi_pagination.bases import AbstractPage
+from models_library.resource_tracker import ContainerListAPI
+from pydantic import PositiveInt
+
+from ..models.pagination import LimitOffsetPage, LimitOffsetParamsWithDefault
+from ..services import resource_tracker_container_service
+
+logger = logging.getLogger(__name__)
+
+
+router = APIRouter()
+
+
+@router.get(
+    "/usage/containers",
+    response_model=LimitOffsetPage[ContainerListAPI],
+    operation_id="list_containers",
+    description="Returns a list of tracked containers for a given user and product",
+)
+async def list_containers(
+    page_params: Annotated[LimitOffsetParamsWithDefault, Depends()],
+    containers_info: tuple[list[ContainerListAPI], PositiveInt] = Depends(
```

Comment:
- Use `Annotated` annotation in `containers_info` as well
- Structuring the values returned by `resource_tracker_container_service.list_containers` would make the code more readable


E.g. 
```python
from typing import NamedTuple

class ContainersPage(NamedTuple): 
    items: list[ContainersListAPI]
    total: PositiveInt


async def list_containers(
    page_params: Annotated[LimitOffsetParamsWithDefault, Depends()],
    containers_page:  Annotated[ContainersPage, Depends(resource_tracker_container_service.list_container) ]
):
    return create_page(
        containers_page.items,
        total=containers_page.total,
        page_params = page_params,
    )
```

NOTE: Doesn't `create_page` complain (with a warning) about not using key arguments? I kind of remember that.

Reply:
Yes, when you add item = "" he complains, but other than that this is ok:
```
    page = create_page(
        containers_info[0],
        total =containers_info[1],
        page_params = page_params,
    )
```